py/mattak/backends/pyroot/mattakloader.py

```diff
@@ -26,25 +29,34 @@ def silent_load(what):
 else:
     libmattakName = 'libmattak.so'
 
+try:
+    import ROOT
+    ds = ROOT.mattak.Dataset()
+    logger.debug("Found libmattak to be already loaded.")
+    loaded = True
+except:
+    pass
+
+
```

Comment:
I always try to avoid double new lines - just for the future

Reply:
rather than instantiating a dataset, it might be lighter weight to do something like

`if ROOT.mattak is not None:  #will throw an attribute error if not loaded`